Update release notes for v2.12 (#8617)
grafana/mimir: docs/sources/mimir/release-notes/v2.12.md

  These counts are more reactive to ring and shard changes than in-memory series, and can be used when enforcing tenant series limits by enabling the `-ingester.use-ingester-owned-series-for-limits` CLI flag.
  This feature requires [zone-aware replication](https://grafana.com/docs/mimir/latest/configure/configure-zone-aware-replication/) to be enabled, and the replication factor to be equal to the number of zones.

- **Support for UTF-8 in Alertmanager** can be enabled via the `-alertmanager.utf8-strict-mode-enabled` CLI flag.

>   When enabled, Alertmanager will be able to receive alerts with labels containing UTF-8 characters,

Reviewer: ```suggestion
  When enabled, Alertmanager can receive alerts with labels containing UTF-8 characters
```

>   and match these alerts in routes, inhibition rules, and silences. This feature has a number of

Reviewer: ```suggestion
  and match these alerts in routes, inhibition rules, and silences. This feature has
```
